dev/parm/config/gcafs/config.fcst

```diff
@@ -57,7 +57,7 @@ case ${RUN} in
     echo "FATAL ERROR: Unsupported RUN '${RUN}'"
     exit 1
 esac
-
+export MERRA2_6ym=".false."
```

Comment:
Consider adding a comment to give a concise description of this variable. In this config and the others.

Also please add a space after this variable line. Check the other configs to add a space as well. Thanks!